src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregatorBase.scala

```diff
@@ -0,0 +1,296 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package com.mozilla.telemetry.streaming
+
+import com.mozilla.telemetry.heka.{Dataset, Message}
+import com.mozilla.telemetry.pings.{CorePing, CrashPing, MainPing, Ping}
+import com.mozilla.telemetry.timeseries.{RowBuilder, SchemaBuilder}
+import org.apache.spark.sql.{DataFrame, Row, SparkSession}
+import org.apache.spark.sql.catalyst.encoders.RowEncoder
+import org.apache.spark.sql.functions.{col, sum, window}
+import org.apache.spark.sql.types.{BinaryType, StructField, StructType}
+import org.json4s.DefaultFormats
+import org.rogach.scallop.{ScallopConf, ScallopOption}
```

Comment:
`ScallopConf` appears to be unused.

Reply:
Note to self: automatically catch unused imports.